### DISCUSSION TITLE
how to import a list of routers

### DISCUSSION THREAD

**Discussion by toloselatrack** [eduardogsilva/routerfleet#34] [Q&A]
hi guys

the program is pretty good

I need to add a bunch of routers (100 or more), can I do it with a sql query?
I have a csv file with all the routers info so I can manage to import to the db but I don't know how to do it

take care

Luca

**Comment by eduardogsilva** [comment]
Hello @toloselatrack !
I have planned to implement an import tool, I just need to finish something else and I will add this tool to routerfleet. If you can wait 2-3 days. I will soon implement this.

Cheers

**Comment by eduardogsilva** [comment]
Hello @toloselatrack 
I just released the import tool. Please let me know how it went for you.
https://github.com/eduardogsilva/routerfleet/discussions/35

Cheers!

**Reply by toloselatrack** [reply]
thank you
going to check it soon!
